docs/source/essentials/01-errors.mdx

```diff
@@ -103,7 +103,9 @@ if (person != null) {
 }
 ```
 
-If you don't want to handle partial responses, you can use `dataOrThrow()`. This returns a non-nullable `data` and throws if there are any errors:
+## Ignoring partial data
+
+If you don't want to handle partial responses, you can simplify your error handling logic.`ApolloResponse.dataOrThrow` will return a non-nullable `data` and throws if there are any errors so that you can handle all eroors in a single `catch {}` block. :
```

Comment:
```suggestion
If you don't want to handle partial responses, you can simplify your error handling logic. `ApolloResponse.dataAssertNoErrors` will return a non-nullable `data` or throw if there are any errors so that you can handle all errors in a single `catch {}` block:
```

Reply:
Spelling mistakes aside, this was made on purpose so that we can merge this PR before we release beta04. The `dataAssertNoErrors` documentation changes are done in https://github.com/apollographql/apollo-android/pull/3510/commits/8a66bbdbaec3fac64c5a6c7864d1895a53147ea2

Reply:
Oh I see! Makes sense. Yeah having the doc published before the new versions is tricky! 

But other instances of `dataOrThrow` were updated as part of this PR 🤔